### DISCUSSION TITLE
v6.3.0

### DISCUSSION THREAD

**Discussion by tay1orjones** [IBM/plex#520] [General]
### Added
- glyphs /acaron /acaron.alt01 /icaron /ocaron /ucaron /udieresismacron /udieresisacute /udieresisgrave /udieresiscaron /Acaron /Icaron /Ocaron /Ucaron /Udieresismacron /Udieresisacute /Udieresisgrave /Udieresiscaron to support pinyin
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
- glyph /longs
  referenced in [issue 158](https://github.com/IBM/plex/issues/158)
- glyph /divisionslash
- old-style numerals
  referenced in [issue 418](https://github.com/IBM/type/issues/418)
- U+2236 RATIO
  referenced in [issue 199](https://github.com/IBM/plex/issues/199)
- U+2219 BULLET OPERATOR 
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
- U+2010-U+2015 NON-BREAKING HYPHEN, DASHES, etc.
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 317](https://github.com/IBM/plex/issues/317)
- U+2028 LINE SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+2029 PARAGRAPH SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+202F NARROW NO-BREAK SPACE
  referenced in [issue 282](https://github.com/IBM/plex/issues/282)
- U+2008 PUNCTUATION SPACE
- U+200A HAIR SPACE
  referenced in [issue 361](https://github.com/IBM/plex/issues/361)
- U+200B ZERO WIDTH SPACE
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 318](https://github.com/IBM/plex/issues/318)

### Fixed
- FamilyClass value in OS/2 table
  referenced in [issue 330](https://github.com/IBM/plex/issues/330)
- STAT table; Roman and Italic styles are correctly linked in various environments now
- incorrect behaviour in Adobe InDesign
  referenced in [issue 337](https://github.com/IBM/plex/issues/337)
- inconsistencies in vertical alignment
  referenced in [issue 377](https://github.com/IBM/plex/issues/377)

### Changed
- replaced commaaccent with cedilla in glyphs /Tcedilla /tcedilla
  referenced in [issue 251](https://github.com/IBM/plex/issues/251)
- disabled bit 38 (Mathematical Operators) from OS/2 UnicodeRanges
- improved hinting

<hr /><em>This discussion was created from the release <a href='https://github.com/IBM/plex/releases/tag/v6.3.0'>v6.3.0</a>.</em>

**Comment by rsms** [comment]
Are there variable fonts available? I only see static font files in the ttf and web zip archives.

**Reply by kenmcd** [reply]
Only for IBM Plex Sans.
Download the source file (plex-6.3.0.zip, ~550MB).
Look in the `IBM-Plex-Sans-Variable` folder.
- IBMPlexSansVar-Italic.ttf
- IBMPlexSansVar-Roman.ttf

**Comment by farahelmasri** [comment]
Hello,

i have an issue with the arabic letters, one letter is missing "K".
![Kaf](https://github.com/IBM/plex/assets/154777706/fbd6cc0e-cc5f-4895-8ed6-0b96d2badf2d)

The letter we want exist in the image i share but needs to be adjusted.

•	The letter “K” in Arabic is present I added a bleu square around it
•	We need to adjust the end of it (on the left side) to resemble to the one in red square
how can we do this?
